tools/configure.cpp

```diff
@@ -0,0 +1,75 @@
+#include <ros/console.h>
+#include <ros/init.h>
+#include <ros/node_handle.h>
+#include <ros/service_client.h>
+
+#include <ed_msgs/Configure.h>
+
+#include <tue/config/configuration.h>
+#include <tue/config/loaders/yaml.h>
+#include <tue/config/resolve_config.h>
+
+#include <tue/filesystem/path.h>
+
+// ----------------------------------------------------------------------------------------------------
+
+void usage()
+{
+    std::cout << "Usage: configure CONFIG_FILE.yaml/json" << std::endl;
+}
+
+// ----------------------------------------------------------------------------------------------------
+
+int main(int argc, char **argv)
+{
+    std::vector<std::string> myargv;
+    ros::removeROSArgs(argc, argv, myargv);
+    if (myargv.size() != 2)
+    {
+        usage();
+        return 1;
+    }
+
+    ros::init(argc, argv, "ed_configure");
+
+    ros::NodeHandle nh;
+    ros::ServiceClient client = nh.serviceClient<ed_msgs::Configure>("ed/configure");
+
+    tue::filesystem::Path config_file(myargv[1]);
+    if (!config_file.exists())
+    {
+        ROS_ERROR_STREAM("Could not configure ED: config file '" << config_file.string() << "' does not exist");
+        return 1;
+    }
+
+    tue::config::ResolveConfig resolve_config;
+    resolve_config.file = false;
+    resolve_config.rospkg = false;
```

Comment:
I would explicitly state the other element of config as true. You dont want this behaviour to change when the default arguments of resolveConfig changes

Reply:
That would be a breaking API, that shouldn't happen that quickly. But definitely the safe option.